app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1297,6 +1298,8 @@ class BrowserTabFragment :
                 if (isActiveCustomTab()) {
                     webView?.hitTestResult?.extra?.let { data ->
                         webView?.loadUrl(Uri.parse(data).toString())
+                    } ?: run {
+                        (activity as CustomTabActivity).openMessageInNewFragmentInCustomTab(it.message, this, customTabToolbarColor)
```

Comment:
In the Custom Tab, there is a single tab, yet links might trigger an "open in new tab" action, requiring us to manage this scenario.